scripts/verify-vocabulary.test.sh

```
@@ -0,0 +1,150 @@
+#!/usr/bin/env bash
+# Negative and positive tests for the vocabulary gate (bash twin of
+# verify-vocabulary.test.ps1): every banned form is rejected in both
+# languages, every pre-registered exemption is honored, malformed manifests
+# and missing scan targets fail loud, and the repository's own document
+# surface passes the gate. A gate only guards if the regression actually
+# fails it.
+
+LC_ALL=C
+cd "$(dirname "${BASH_SOURCE[0]}")/.." || exit 1
+source scripts/lib.sh
```

Shellcheck (0.11.0)

[info] Not following: scripts/lib.sh was not specified as input (see shellcheck -x).

(SC1091)